src/main/java/com/example/ludogoriesoft/lukeriaerpapi/services/PackageService.java

```diff
@@ -2,98 +2,98 @@
 
 
 import com.example.ludogoriesoft.lukeriaerpapi.dtos.PackageDTO;
-import com.example.ludogoriesoft.lukeriaerpapi.exeptions.ApiRequestException;
-import com.example.ludogoriesoft.lukeriaerpapi.mappers.PackageMapper;
 import com.example.ludogoriesoft.lukeriaerpapi.models.Package;
+import com.example.ludogoriesoft.lukeriaerpapi.repository.CartonRepository;
 import com.example.ludogoriesoft.lukeriaerpapi.repository.PackageRepository;
 import io.micrometer.common.util.StringUtils;
+import jakarta.validation.ValidationException;
 import lombok.AllArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.data.crossstore.ChangeSetPersister;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
-import java.util.Optional;
 
 @Service
 @AllArgsConstructor
 public class PackageService {
     private final PackageRepository packageRepository;
-    private final PackageMapper mapper;
-
-    public PackageDTO toDTO(Package packageEntity) {
-        return mapper.toDto(packageEntity);
-    }
-
-    public Package toEntity(PackageDTO packageDTO) {
-        return mapper.toEntity(packageDTO);
-    }
-
+    private final CartonRepository cartonRepository;
+    private final ModelMapper modelMapper;
 
     public List<PackageDTO> getAllPackages() {
-        List<Package> packages = packageRepository.findAll();
-        return packages
-                .stream()
-                .map(this::toDTO)
-                .toList();
+        List<Package> laptops = packageRepository.findByDeletedFalse();
+        return laptops.stream().map(package1 -> modelMapper.map(package1, PackageDTO.class)).toList();
```

Comment:
package1 should can be called just package

Applicable for all places.

Reply:
But ''package'' is a reserved word in java language

Reply:
You're right. Sorry for the confusion